addons/account/views/res_config_view.xml

```diff
@@ -273,6 +273,9 @@
                     <group string="Overdue Payments" name="followup">
                         <field name="overdue_msg"/>
                     </group>
+                    <group id="edi_group" string="Electronic Data Interchange" name="followup">
+                        <field name="edi_xml_attached_mails"/>
+                    </group>
```

Comment:
This change is required to fix the red of https://github.com/odoo/enterprise/pull/660